script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -296,27 +318,27 @@ Senpai?!
 [sha:bc620dadbd13115f88b4327647d216c883bc1727]{
 -- Page 375, Offset 28088.
 -- 「―――確かめに行きます。
-"I'm going there to ascertain this.
+"I need to go verify this.
 }
 [sha:13f564a97af6df2e393296e9fcaf23f9e7ad1f5a]{
 -- Page 375, Offset 28089.
 -- 遠野くんはここに残っていてください」
-Please stay right here, Tohno-kun."
+Please don't leave this room, Tohno-kun."
```

Comment:
I don't really see the need for the change.

Reply:
Something felt off about original to me. I guess to put a finger on it, "stay right _here_" feels like something you'd say to tell someone to stay _with_ you somewhere. I could just change it to "stay right there" or "stay right where you are", but they felt somehow more restrictive? As in to not leave the exact spot he's sitting in, rather than the general area. Very minor interpreted nuance stuff.

Reply:
Maybe the pleast is a bit restrictive? I dunno. Random alternative:
>I'd appreciate it if you stayed put, Tohno-kun.

Reply:
Works for me